### PR TITLE
Bugfix/player 3308

### PR DESCRIPTION
**Pull request by yaroslavgribov**
Fixes issue where user selected default track was set after SET_CURRENT_AUDIO rather than actual user input.

### CODE REVIEW

**Comment by pilievOoyala** [comment]
LGTM

File: js/controller.js [hunk 1]
```diff
@@ -897,6 +897,10 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
      * @param currentTrack {{id: String, label: String, lang: String}} - current active audio track
      */
     setCurrentAudio: function(currentTrack) {
+      // user selected track should be saved to localstorage
+      var currentAudioTrack = JSON.stringify(currentTrack);
+      OO.setItem(OO.CONSTANTS.SELECTED_AUDIO, currentAudioTrack);
```
Comment (aeng7) [inline js/controller.js]: Where is the stored value used? Just out of curiosity

**Comment by aeng7** [comment]
LGTM other than a small comment

File: js/controller.js (hunk 1 above)
Comment (yaroslavgribov) [reply js/controller.js]: @aeng7 Thanks for the review. 
We use it later in the core to restore the default track based on user previous selection.

**Pull request merged**